modules/pytket-braket/pytket/extensions/braket/backends/braket.py

```diff
@@ -625,6 +670,58 @@ def characterisation(self) -> Optional[Dict[str, Any]]:
     def backend_info(self) -> BackendInfo:
         return self._backend_info
 
+    @classmethod
+    def available_devices(cls, **kwargs: Any) -> List[BackendInfo]:
+        """
+        See :py:meth:`pytket.backends.Backend.available_devices`.
+        Supported kwargs: none.
+        """
+        braket_regions = ["us-west-1", "us-west-2", "us-east-1"]
```

Comment:
These have to be hardcoded because there isn't an obvious way to fetch available regions for the `braket` service. The `get_available_regions` method in `boto3` returns an empty list for `braket`. I assume these regions should be quite stable.

Reply:
OK, thanks. I would revert to how it was before -- feels weird to hard-code them, even if they are _likely_ to be stable -- not sure if this would work in all situations, e.g. using from China? Let's leave the choice of region up to the user if they want to specify it.

Reply:
Yeah, I agree